src/App.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+
+import {Home} from './components/HomeScreen';
+import Placeholder from './components/PlaceholderScreen';
+
+import NavUtils from './utils/NavUtils';
+
+const Stack = createNativeStackNavigator();
+
+interface State {}
+
+export default class App extends React.Component<{}, State> {
+  
+  constructor(props: any) {
+    super(props);
+
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <NavigationContainer>
+        <Stack.Navigator
+          defaultScreenOptions={{
+            headerTintColor: 'white',
+          }}
+          screenOptions={{
+            presentation: 'transparentModal'
```

Comment:
Before adding this, when a new screen is being navigated to, on Android, I was seeing a flash of white behind the new screen animating in.  By using this transparentModal setting it went away, I suppose since the black background of previous screen is allowed to show through.

Reply:
Weird. 

https://stackoverflow.com/questions/59900898/white-background-flashing-when-switching-screens-react-navigation-v5

Look like we can add a `View` wrapper around Navigation to resolve this. Would you kindly address this in the PR?

- [ ] Update with wrapper around `Navigation` component

Reply:
Good find @garyray-k - this is done now, works well and looks better.